.github/workflows/codeql-analysis.yml

```diff
@@ -65,4 +65,4 @@ jobs:
     #   make release
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v1
\ No newline at end of file
+      uses: github/codeql-action/analyze@v1
```

Comment:
I believe there's a v2 for the analyze workflow, as well as the init and autobuild tasks.